### DISCUSSION TITLE
Ability to search Dashboard and Panel in annotations

### DISCUSSION THREAD

**Discussion by gtirloni** [grafana/grafana#73525] [Alerting]
Alerts have two annotations `Dashboard UID` and `Panel ID` that are used to link to to dashboards and panels.

One has to manually find these numbers to link a standalone alert to them.

The ability to search them from the Edit Alert page would be a great improvement.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82573. Feel free to continue the discussion around this there. Thank you!
